jobs/cache_maintenance/src/cache_maintenance/cache_metrics.py

```diff
@@ -0,0 +1,29 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2023 The HuggingFace Authors.
+
+import logging
+
+from libcommon.simple_cache import (
+    CacheTotalMetricDocument,
+    get_responses_count_by_kind_status_and_error_code,
+)
+
+
+def collect_cache_metrics() -> None:
+    logging.info("collecting cache metrics")
+    for metric in get_responses_count_by_kind_status_and_error_code():
+        kind = metric["kind"]
+        http_status = metric["http_status"]
+        error_code = metric["error_code"]
+        new_total = metric["count"]
+
+        query_set = CacheTotalMetricDocument.objects(kind=kind, http_status=http_status, error_code=error_code)
+        current_metric = query_set.first()
+        if current_metric is not None:
+            current_total = current_metric.total
+            logging.info(
```

Comment:
excellent, it would be interesting to see these logs: do the metrics diverge, or does it work well?